env/includes/settings/apps/tk-multi-workfiles2.yml

```diff
@@ -88,9 +88,11 @@ settings.tk-multi-workfiles2.task: &default_workfiles2_task_settings
 # houdini task
 settings.tk-multi-workfiles2.houdini.task:
   <<: *default_workfiles2_task_settings
+  hook_scene_operation: '{config}/tk-multi-workfiles2/scene_operation_{engine_name}.py'
```

Comment:
I'm not sure what is being overridden here, but if only some methods are being overriden in this file:
* copy only those methods in config and modify them
* use hook inheritance (eg. see above tk-multi-loader2) `self/scene_operation_{engine_name}.py:{config}/tk-multi-workfiles2/scene_operation_{engine_name}.py`

Reply:
There is only one method in the base hook and its being overridden.